connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/inbound/InboundConnectorContextImpl.java

```diff
@@ -101,6 +96,11 @@ public InboundConnectorContextImpl(
         logs);
   }
 
+  @Override
+  public ActivationCheckResult canActivate(Object variables) {
```

Comment:
What about `ActivationResult`?

Reply:
We discussed that with the team and came up with this name, as we are not activating anything. `ActivationResult` might be a bit misleading in that regard

Reply:
I see, although we activate without correlate. 😄